docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     command: ./manage.py celery worker --events --beat --autoreload --concurrency=4 -Q mdn_purgeable,mdn_search,mdn_emails,mdn_wiki,celery
     user: ${UID:-33}
     volumes:
-      - ./:/app
+      - ./:/app:z
```

Comment:
From [the docs](https://docs.docker.com/engine/admin/volumes/bind-mounts/#configure-the-selinux-label):

> If you use selinux you can add the z or Z options to modify the selinux label of the host file or directory being mounted into the container
> The z option indicates that the bind mount content is shared among multiple containers.
> The Z option indicates that the bind mount content is private and unshared.